src/sc_crawler/schemas.py

```diff
@@ -49,8 +50,15 @@ def __init__(subclass, *args, **kwargs):
             comment = satable.columns[k].comment
             if v.description and comment is None:
                 satable.columns[k].comment = v.description
-        # append inserted_at as last column
-        satable.append_column(Column("inserted_at", DateTime, default=datetime.utcnow))
+        # append observed_at as last column
```

Comment:
Will/are we get/ting the actual version of the DB before running this, or will this column always contain new values on every run?